Gemfile

```diff
@@ -4,13 +4,15 @@ ruby File.read(".ruby-version").strip
 
 source "https://rubygems.org"
 
-gem "rails", "~> 6.0.2"
 
 gem "bootsnap", "~> 1"
 gem "dynamoid"
 gem "govuk_app_config", "~> 2.1.1"
 gem "govuk_publishing_components", "~> 21.34.0"
 gem "puma", "~> 4.3"
+gem "rails", "~> 6.0.2"
+gem "redis"
+gem "redis-rails", "~> 5.0"
```

Comment:
Do we need this? Seems to be in-built - https://github.com/redis-store/redis-rails#a-quick-note-about-rails-52?

Reply:
I wonder if that also applies to sessions? I'm using that gem for the session stuff:

> We are still actively maintaining all other gems in the redis-store family, such as redis-actionpack for session management

Reply:
Yeah, it's not clear from the documentation. I guess if this works, we can look at it later, if its an issue!